diverter_position: 1.57
amp_position: 0.78
trap_position: 1.05
tolerance: 1